src/pkg/backup/backup.go

```diff
@@ -111,12 +112,21 @@ func New(
 		}
 	}
 
+	tags := map[string]string{
+		model.ServiceTag: selector.PathService().String(),
+	}
+
+	// Add tags to mark this backup as either assist or merge. This is used to:
+	// 1. Filter assist backups by tag during base selection process
+	// 2. Differentiate assist backups from merge backups
+	if isAssistBackup {
+		tags[model.AssistBackupTag] = ""
+	}
```

Comment:
it might actually be easier to use the same key (e.x. "status" or something) and add a value as either "assist-base" or "merge-base" (naming TBD, there for example purposes)

The reasoning for this is that it can make selecting backups to display to the user and/or sanity checking a bit easier. We can't switch to nice clean logic that operates on one or the other tag yet since there's still old backups, but if we ever break backwards compatibility it'll give us a leg up. For sanity checking, it helps enforce the idea that everything will be tagged and everything is highly likely to have only one or the other tag because operators like `+=` are unlikely to be used for a single field